app/templates/components/add-env-var.hbs

```diff
@@ -1,21 +1,48 @@
-<form {{action (perform save) on='submit'}}>
-  <div class="form-elem">
-    {{input value=name class="env-name" key-up=(action 'nameChanged') placeholder="Name"}}
-    {{#if nameIsBlank }}
-      <p class="form-error-message">Name cannot be blank</p>
-    {{/if}}
+{{#travis-form
+  onSubmit=(perform save)
+  as |form|
+}}
+  <div class='form-elem'>
+    {{#form.field
+      label='Name'
+      value=name
+      required=true
+      showValidationStatusIcons=false
+      as |field|
+    }}
+      {{field.input class='env-name' placeholder='Name' onChange=(action (mut name))}}
```

Comment:
`onChange` should be bound to `form.field`, not to `field.input`, in order to handle all states and validations properly.